### ISSUE TITLE
Feature Request - Colour Code Lists/Caches

### ISSUE THREAD

**Issue by yellowduck** [cgeo/cgeo#3523]
I keep many lists to separate out caches into groups. Examples include Church Micros (a very popular series in the UK), specific D/T ratings, date hidden, interesting caches, night caches, etc. When out and about, I use the 'enable live' feature to display these on the map, however, once the map becomes populated, it is very difficult to distinguish the location of these 'special'  caches amongst all the others.
(Note, I also have lists for letterboxes and virtuals, which are easier to see due to the distinctive icons).

My proposal is to be able to apply colour codes to either complete lists or individual caches in order to highlight them as priority targets.

The c:geo icon for a cache has a border of white around it (grey for a disabled or archived cache). The intention is to be able to change this border - only a small number of primary colours would be needed. See mock-up on the attched image.

![2014-01-07 13 23 26](https://f.cloud.github.com/assets/1512344/1859231/14696af8-779f-11e3-811f-2709768870d6.png)

**Comment by yellowduck** [comment]
This has been re-visited with #5798

**Comment by yellowduck** [comment]
Also similar #5037

**Comment by Kjerkloe** [comment]
I just wanted to propose a similar feature and I saw this was already suggested. 

I travel often, and every time I visit a cache-dense area, I'm missing a similar feature suggested here. It would be really usefull to have some sort of tool to mark or highlight certain caches on the map. It doesn't have to be over complicated, but even one distinctive color or any other feature that would make user marked caches (or a list of  caches) to stand out from the others, would make a huge difference and would make this awesome app even more usefull... 

Is there any progress on this topic?

**Comment by magma1447** [comment]
I have for years wanted pretty much the exact same thing as the above request.

Use case example number 1:
I might have lists of D/T caches that I need. If I happen to drive fairly close to them I might want to take that detour. It therefore helps to know what it is by having it color coded, to not confuse it with all other lists one might have stored in the app.

Use case example number 2:
If I go with a group to geocache for the weekend, we usually have a shared list with our primary target of geocaches. This is stored as an offline list. Added to that, I might want to have 1-2 lists myself, that I don't feel is so important, but if we are in the vicinity I would like the group to take that small detour. This might be D/T, hidden month, a mystery I solved or whatever.

My idea has been exactly the same as the original post here. Color code lists, by changing the background color. I understand that it might conflict with the grey/disabled. In my opinion, color coded lists could/should have presence.

Ultimately, I would also like to be able to turn on/off offline lists. So that I can choose which offline lists are shown on the map, and which aren't.
